src/ImageSharp/ColorSpaces/CieLch.cs

```diff
@@ -83,11 +78,7 @@ public CieLch(Vector3 vector, CieXyz whitePoint)
         /// <summary>
         /// Gets the reference white point of this color
         /// </summary>
-        public CieXyz WhitePoint
-        {
-            [MethodImpl(MethodImplOptions.AggressiveInlining)]
-            get;
-        }
+        public CieXyz WhitePoint { get; }
```

Comment:
I suppose autoproperty code is always inlined by the JIT, no need for explicit requests.

Reply:
Aye good thinking, that's likely the case (would like to check sometime though to be certain). Colorspace code is performing well anyway.

Reply:
Methods with IL code shorter than 32 bytes are usually inlined [even in old .NET frameworks](https://blogs.msdn.microsoft.com/davidnotario/2004/11/01/jit-optimizations-inlining-ii/). Heuristics got more sophisticated since then.